test/cases/amd/namedModules/index.js

```diff
@@ -0,0 +1,30 @@
+define("named1", [], function() {
+	return "named1";
+});
+
+define("named2", [], function() {
+	return "named2";
+});
+
+define("named3", [], function() {
+	return "named3";
+});
+
+define("named4", [], function() {
+	return "named4";
+});
+
+define(["named1", "named2"], function(named1, named2) {
+	it("should load the named modules in defined dependencies", function() {
+		"named1".should.be.eql(named1);
```

Comment:
Could you switch variable and string so the `expected` and `actual` values are correct?

``` js
named1.should.be.eql("named1");
```

Reply:
OK.  The only reason I did it that way was to avoid throwing an exception if the variable named1 is undefined (e.g. "Unable to read property should of undefined").

Reply:
BTW, could you provide pointers to webpack-bot info?  I like the review workflow that you've implemented and would like to do something similar with projects I work on.  Thanks.

Reply:
https://github.com/open-bot/open-bot

and

https://github.com/webpack/webpack/blob/master/open-bot.yaml